modules/statistics/test/TestPlan.md

```diff
@@ -25,7 +25,7 @@
 11. On the incomplete double data entry page, check that the candidate links take you to the appropriate data entry page for that candidate
 12. Returning to the behavioral statistics tab, check that the "Click here for breakdown per participant for" link below the Double Data Entry Statistics table works.
 13. Verify that this link takes you to page with double data entry statistics for all sites
-14. Returning to the behavioural statistics page, verify that if you do not have the access_all_profiles permission you should not be able to click through to forms for candidates from other sites.
+14. Returning to the behavioural statistics page, verify that if you do not have the `access_all_profiles` permission you should not be able to click through to forms for candidates from other sites.
 15. Check the the breadcrumbs from Completion stats page back to BVL stats tab within Statistics appear and allow you to use the `Back` button to previous pages (future feature).
```

Comment:
Check the breadcrumbs.

In section Imaging, put quote around:
line 3 'Select All'
line 4 'Breakdown by Scan Type'
line 5 'show/hide'

Reply:
good catch! i'll make these changes